example/docker-compose.yml

```diff
@@ -2,7 +2,7 @@ version: '2'
 services:
   cke:
     container_name: cke
-    image: quay.io/cybozu/cke:1.21.1
+    image: quay.io/cybozu/cke:1.22.9
```

Comment:
Set the latest tag. When after releaseing v1.23.0, I will update this tag.
https://github.com/cybozu-go/cke/blob/v1.22.9/.github/workflows/main.yaml#L118-L122